<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>等线</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Data Analysis and Visualization of Hangzhou Metro Traffic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>810</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>960</width>
        <height>730</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>20</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>30</y>
        <width>100</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>70</y>
        <width>100</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>lineID</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>110</y>
        <width>100</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>stationID</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>150</y>
        <width>100</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>deviceID</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>190</y>
        <width>100</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>status</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>230</y>
        <width>100</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>userID</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>270</y>
        <width>100</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>payType</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>1370</x>
        <y>110</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1480</x>
        <y>109</y>
        <width>93</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>1370</x>
        <y>70</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>1480</x>
        <y>68</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>1370</x>
        <y>190</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>0(out)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_9">
      <property name="geometry">
       <rect>
        <x>1480</x>
        <y>190</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>1(in)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>1350</x>
        <y>300</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Display All</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Trend</string>
     </attribute>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>80</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QChartView" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>1301</width>
        <height>731</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd hh:mm:ss</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Station</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting Time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ending Time</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd hh:mm:ss</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Time Step</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>mins</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Route</string>
     </attribute>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>31</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>80</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>31</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>80</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>Entering Station ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>30</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Exiting Station ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>1541</width>
        <height>661</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Design Routes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>810</y>
      <width>280</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
     </font>
    </property>
    <property name="text">
     <string>Honor-produced by Yuxiang Lu @2019</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>830</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
     </font>
    </property>
    <property name="text">
     <string>All rights reserved</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>810</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>810</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>26</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
